test/test_fx_experimental.py

```diff
@@ -1480,6 +1480,7 @@ def test_normalize_operator_exhaustive(self, device, dtype, op):
             "igammac",
             "index_put",
             "nn.functional.conv2d",
+            "nn.functional.alpha_dropout",
```

Comment:
Can this test be skipped (or better, xfailed) using a DecorateInfo on the OpInfo?

cc @Chillee

Reply:
This has been added as an expected failure, also for other dropout variants. Thanks for the suggestion!